src/models/room.js

```diff
@@ -1866,8 +1866,7 @@ function calculateRoomName(room, userId, ignoreRoomNameEvent) {
 
     const joinedMemberCount = room.currentState.getJoinedMemberCount();
     const invitedMemberCount = room.currentState.getInvitedMemberCount();
-    // -1 because these numbers include the syncing user
-    const inviteJoinCount = joinedMemberCount + invitedMemberCount - 1;
```

Comment:
This was actually a bug it seems. `inviteJoinCount` is only used in the changed if statement below and for passing to `memberNamesToRoomName`, but `memberNamesToRoomName` expects the count to be including the current user.